specification/search/data-plane/Azure.Search/stable/2023-11-01/searchservice.json

```
@@ -5530,6 +5530,207 @@
       },
       "description": "Ranking function based on the Okapi BM25 similarity algorithm. BM25 is a TF-IDF-like algorithm that includes length normalization (controlled by the 'b' parameter) as well as term frequency saturation (controlled by the 'k1' parameter)."
     },
+    "VectorSearch": {
+      "type": "object",
+      "properties": {
+        "profiles": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/VectorSearchProfile"
+          },
+          "description": "Defines combinations of configurations to use with vector search."
+        },
+        "algorithms": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/VectorSearchAlgorithmConfiguration"
+          },
+          "description": "Contains configuration options specific to the algorithm used during indexing and/or querying."
+        }
+      },
+      "description": "Contains configuration options related to vector search."
+    },
+    "VectorSearchProfile": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "externalDocs": {
+            "url": "https://docs.microsoft.com/rest/api/searchservice/Naming-rules"
```

We should probably use the learn. subdomain now.

```suggestion
            "url": "https://learn.microsoft.com/rest/api/searchservice/Naming-rules"
```